ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -1667,3 +1668,66 @@ export const testFlex = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testRichTextEditor = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      let editedText = '';
+      const components: Component[] = [
+        {
+          id: 'RTE1',
+          key: 'RTE1',
+          type: WhisperComponentType.RichTextEditor,
+          onBlur: () => {
+            console.debug(`On blur called`);
+          },
+          onChange: (_error: Error, value: string) => {
+            console.debug(`Input value changed: ${value}`);
+            editedText = value;
+          },
+          onFocus: () => {
+            console.debug(`On blur called`);
+          },
+          tooltip: "It's a richTextEditor tooltip.",
+        },
+        {
+          type: WhisperComponentType.Box,
+          justifyContent: JustifyContent.Right,
+          direction: Direction.Horizontal,
+          children: [
+            {
+              type: WhisperComponentType.Button,
+              label: 'Save',
+              onClick: (_error: Error, onClickWhisper: Whisper) => {
+                if (!editedText || editedText.length === 0 || editedText.length > 50) {
+                  (components[0] as RichTextEditor).validationError =
+                    'Inputed text is required and should be less than 200 chars.';
+                  onClickWhisper.update({
+                    components,
+                  });
+                } else {
+                  (components[0] as RichTextEditor).validationError = null;
```

Comment:
This line is not needed as we are not using components bellow. It is probably my oversight, please remove it ;)